doc/assets.md

```diff
@@ -10,41 +10,37 @@ For assets, a special **`ezdesign` asset package** is available.
 <img src="{{ asset("images/foo.png", "ezdesign") }}" alt="foo"/>
 ```
 
-Using `ezdesign` package will resolve current design with theme fallback.
+Using the `ezdesign` package will resolve current design with theme fallback.
 
 By convention, an asset theme directory can be located in:
-* `<bundle_directory>/Resources/public/themes/`
-* `web/assets/themes/`
+- `<bundle_directory>/Resources/public/themes/`
```

Comment:
> would make maintaining and editing the doc easier.

Consistency across everything doesn't really make editing and maintaining easier here. What matters is that the same one is used over documents, I think. I'm more okay with that one than with the line length, though.

Just remember that any change is logged, and doing that makes you the last author of the change, making it more complicated to trace down which change a line came from. If after this change somebody does a blame on the file, your change will displayed, while it is not when the content was introduced.

If consistency on bullet points is important, then it should be written down somewhere, and it should be easy to find for doc contributors / developers.